roles/installer/defaults/main.yml

```diff
@@ -223,7 +226,12 @@ postgres_storage_requirements:
   requests:
     storage: 8Gi
 postgres_init_container_resource_requirements: {}
-postgres_resource_requirements: {}
+postgres_resource_requirements:
```

Comment:
@fosterseth I noticed this PR after putting up this other PR that makes a similar change: https://github.com/ansible/awx-operator/pull/876

I think your default request values may be a little too high (requests are guaranteed resources, it doesn't prevent the container from using more when needed).

Reply:
Let's sync up next week on how to merge these.  Proposal: if you fold in my changes in #876 and fix the conflicts, we can just merge this and close the other.

Reply:
I pulled your commit into this PR

Reply:
Thank you @fosterseth 

I will pull this down and test it out one more time before merging.